### PR TITLE
Tweaks from 5273

### PR DESCRIPTION
**Pull request by eteq**
This has a few minor changes from #5273 that I would have pointed out in a review, but I was too slow at reviewing it.  Look good to you @StuartLittlefair ? (In particular, note that I changed these to "new feature" and v1.3 in the changelog, as it's definitely a new feature.)

### CODE REVIEW

**Comment by eteq** [comment]
hah, well, I think it really is of general use so it's probably good to be in utils (where someone might conceivably go looking if they're trying to figure out how to do that...)

**Comment by mhvk** [comment]
Yes, that is funny, as I asked @StuartLittlefair to move that routine out of `utils.py` feeling it was just a little too specific...   But no objection to moving it back.

**Comment by eteq** [comment]
Oh, I missed that.  Well, I guess this is my vote, then ;)

File: CHANGES.rst [hunk 1]
```diff
@@ -20,6 +20,9 @@ New Features
     For JPL kernels, this roughly doubles the execution time, so if one requires
     only the positions, one should use ``get_body_barycentric``. [#5231]
 
+  - Transformations between coordinate systems can use the more accurate JPL
+    ephemerides. [#5273]
```
Comment (pllim) [inline CHANGES.rst]: Astrobot is going to complain that #5436 is not listed... Also same with the other entry.

**Comment by eteq** [comment]
Good points, @pllim - hopefully astrobot will will like it now... Here goes!

**Comment by eteq** [comment]
Yay! I didn't make @astrobot angry!

**Comment by pllim** [comment]
It is funny how making `astrobot` happy is now permanently documented under commit history. :smile:

**Pull request merged**